### ISSUE TITLE
Tolino error-message "Die gewünschte Aktion konnte leider nicht ausgeführt werden, da der Titel defekt ist"

### ISSUE THREAD

**Issue by zisco67** [rvolz/BicBucStriim#362]
I get this message, when I'm trying to download a title to (all of) my Tolinos; I have authentication enabled, I'm using HTTPS and I've seen the discussion in #114 - but since this issue is pretty old, I'm wondering, if there is a solution for that problem now?

**Comment by rvolz** [comment]
The HTTP download works as mentioned in #239. I just tested it and it still does.
I tested with a tolino vision 4, software version 14.1.0.

I can't test the HTTPS easily with my setup. Since the protocol should be transparent to the app, are you sure it isn't a certificate issue? Are you using a self-signed certificate?

**Comment by zisco67** [comment]
Yes, it's a self-signed certificate. And either HTTP nor HTTPS are working. How can I dive deeper to see what's happening in detail?

**Comment by rvolz** [comment]
To see if there are server-side problems you could enable [debug mode](https://github.com/rvolz/BicBucStriim/wiki/Debug-mode) in BicBucStriim. 

About the certificates: many networks stacks don't accept self-signed certificates anymore and throw errors. If you can't find errors with debug mode, you could try a [let's encrypt certificate](https://letsencrypt.org/) and see if that helps with https.
